includes/templates/template_default/templates/tpl_modules_checkout_address_book.php

```diff
@@ -18,6 +18,9 @@
 <?php
   foreach ($addresses as $address) {
     $selected = ($address['address_book_id'] == $_SESSION['sendto']);
+    if ($current_page_base == 'checkout_payment_address') {
```

Comment:
That should use `===` and `FILENAME_CHECKOUT_PAYMENT_ADDRESS`.

Reply:
Good catch.